jscomp/others/js_json.ml

```diff
@@ -60,6 +61,35 @@ let test (type a) (x : 'a) (v : a kind) : bool =
   | Array -> Js_array.isArray x 
   | Object -> (Obj.magic x) != Js.null && Js.typeof x = "object" && not (Js_array.isArray x )
 
-
 external parse : string -> t = "JSON.parse" [@@bs.val]
 (* TODO: more docs when parse error happens or stringify non-stringfy value *)
+
+let null : t = (Obj.magic (Js.null: 'a Js.null) : t)
+
+external string : string -> t = "%identity"
+
+external number : float -> t = "%identity"
+
+external numberOfInt : int -> t = "%identity"
```

Comment:
`int` is not part of json standard, let's keep the API simple. you can always 
(float x) to convert it into float